macros/DocStatusQuickLinks.ejs

```diff
@@ -81,40 +81,59 @@ var titleValues = mdn.localStringMap({
         <li><strong><a href="/<%=env.locale%>/docs/MDN/Doc_status/Overview"><%=titleValues['Overview']%></a></strong></li>
         <li><strong><%=titleValues['Sections']%>:</strong></li>
             <ol>
-                <li><a href="/<%=env.locale%>/docs/MDN/Doc_status/Accessibility"><%=titleValues['Accessibility']%></a>
-                <li><a href="/<%=env.locale%>/docs/MDN/Doc_status/API"><%=titleValues['API']%></a>
-                <li><a href="/<%=env.locale%>/docs/MDN/Doc_status/Apps"><%=titleValues['Apps']%></a>
-                <li><a href="/<%=env.locale%>/docs/MDN/Doc_status/CSS"><%=titleValues['CSS']%></a>
-                <li><a href="/<%=env.locale%>/docs/MDN/Doc_status/DevTools"><%=titleValues['DevTools']%></a>
-                <li><a href="/<%=env.locale%>/docs/MDN/Doc_status/Firefox_OS"><%=titleValues['Firefox_OS']%></a>
-                <li><a href="/<%=env.locale%>/docs/MDN/Doc_status/Games"><%=titleValues['Games']%></a>
-                <li><a href="/<%=env.locale%>/docs/MDN/Doc_status/HTML"><%=titleValues['HTML']%></a>
-                <li><a href="/<%=env.locale%>/docs/MDN/Doc_status/JavaScript"><%=titleValues['JavaScript']%></a>
-                <li><a href="/<%=env.locale%>/docs/MDN/Doc_status/Marketplace"><%=titleValues['Marketplace']%></a>
-                <li><a href="/<%=env.locale%>/docs/MDN/Doc_status/MathML"><%=titleValues['MathML']%></a>
-                <li><a href="/<%=env.locale%>/docs/MDN/Doc_status/SVG"><%=titleValues['SVG']%></a>
+                <li><a href="/<%=env.locale%>/docs/MDN/Doc_status/Accessibility"><%=titleValues['Accessibility']%></a></li>
```

Comment:
``{{DocStatusQuickLinks}}`` looks good 👍. Indentation of multi-line summaries is different.

<img width="468" alt="pr 789 docstatusquicklinks" src="https://user-images.githubusercontent.com/286017/45332916-049d9680-b539-11e8-91e5-a61a297a9cf2.png">